### DISCUSSION TITLE
shipment to belgium?

### DISCUSSION THREAD

**Discussion by marc-portier** [micro-nova/AmpliPi#191] [General] [answered]
am considering amplipi 

living in Belgium though - any luck in affordable shipment that can get the goods over here?
anyone with overseas experience ?

**Comment by marc-portier** [comment, marked as answer]
just continued further to the pre-order form and got my answer
found out Belgium is just nicely in the list of countries supported by your shipping providing

sorry for the noise, will report later on effective experience

**Reply by jjjeroenski** [reply]
Hi @marc-portier , 
Did you bought already?
I also ended up on this product last week and was wondering the expected customs costs to Belgium?

**Reply by marc-portier** [reply]
just got it delivered with UPS yesterday

the ups import slip showed extra 225 € (including import VAT, duties, brokerage surcharges)

**Reply by jjjeroenski** [reply]
ok thanks good to know.
good luck with the deliverd purchase!

**Reply by jonathanslenders** [reply]
Just got my UPS invoice, 223.25 € here for Belgium.

**Comment by jonathanslenders** [comment]
In case anyone else from Belgium (or the Netherlands or France) comes across this page. Feel free to reach out to me if you want to try a unit. I'm considering to sell mine, even though I'm super happy with the hardware quality and support, but thinking about building a more advanced Motu/AVB setup. That would save you at least the awful import costs.

**Reply by jjjeroenski** [reply]
hi @jonathanslenders,
do you still have this unit?
i am interested in the unit, but also in the reasons for a more advanced solution. DM?

**Reply by jonathanslenders** [reply]
Hi @jjjeroenski, yes, I still have the unit.

I don't know how to send a DM. You can reach out by mail. My email address you can find on my GitHub homepage: https://github.com/jonathanslenders

**Comment by dries74** [comment]
Hello @marc-portier, @jonathanslenders, can you please check if your AmpliPi units include a [CE marking](https://en.wikipedia.org/wiki/CE_marking) anywhere? Thanks.

**Reply by gorski123** [reply]
Hello @dries74, thanks for joining and thanks for the post! Yes - we shipped our Kickstarter prototype units and pre-order units with the CE mark. While we have completed all the tests internally to verify CE compliance, we were still a bit unaware of the large amounts of documentation that are required to put together a full DoC - something that is required along with the CE mark for full product entry into the EU market. We are still in the process of putting together the report to have a complete DoC (Declaration of Conformity). Until that process is complete, we are not shipping AmpliPi units to EU countries. Sorry very much for the inconvenience. We are working very very hard to overcome this issue, but unfortunately we are a very small company and this process can take a long time and be very expensive to complete correctly. 
Thanks!
Jason
